src/Microsoft.ML.Data/DataLoadSave/Text/TextLoader.cs

```diff
@@ -1283,7 +1283,8 @@ internal static IDataLoader Create(IHostEnvironment env, Arguments args, IMultiS
         internal static IDataView ReadFile(IHostEnvironment env, Arguments args, IMultiStreamSource fileSource)
             => new TextLoader(env, args, fileSource).Read(fileSource);
 
-        public void Save(ModelSaveContext ctx)
+        void ICanSaveModel.Save(ModelSaveContext ctx) => SaveModel(ctx);
+        internal void SaveModel(ModelSaveContext ctx)
```

Comment:
>internal [](start = 8, length = 8)

Can this be private?

Reply:
Yes it can! Good point thank you.

---
In reply to: [254473480](https://github.com/dotnet/machinelearning/pull/2431#discussion_r254473480) [](ancestors = 254473480)